app/modules/claim.js

```diff
@@ -31,7 +31,7 @@ import { getNode, getRPCEndpoint } from '../actions/nodeStorageActions'
 // Constants
 export const DISABLE_CLAIM = 'DISABLE_CLAIM'
 const POLL_ATTEMPTS = 30
-const POLL_FREQUENCY = 10000
+const POLL_FREQUENCY = 4000
```

Comment:
Why do we need to double the polling frequency?

Reply:
Arbitrary UX enhancement.